include/engine/api/nearest_api.hpp

```diff
@@ -110,6 +111,11 @@ class NearestAPI final : public BaseAPI
                                nodes.values.push_back(node_values.second);
                                waypoint.values["nodes"] = std::move(nodes);
 
+                               util::json::Array osm_nodes;
+                               osm_nodes.values.emplace_back(std::to_string(node_values.first));
+                               osm_nodes.values.emplace_back(std::to_string(node_values.second));
+                               waypoint.values["osm_nodes"] = std::move(osm_nodes);
```

Comment:
In the nearest API we unconditionally add this new `osm_nodes` array, i.e. always have kind of duplication.